workspaces/adr/.changeset/sixty-balloons-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/search-backend-module-adr': minor
+---
+
+**BREAKING** Removed `TokenManager` as it has been removed from the upstream Backstage packages. Please use the new backend system.
```

Comment:
Following up on my other comment - lets also note that the `AuthService` is now required to work with the new backend system

Reply:
Done 👍